libraries/AP_Landing/AP_Landing_Deepstall.cpp

```diff
@@ -192,10 +192,24 @@ bool AP_Landing_Deepstall::verify_land(const Location &prev_WP_loc, Location &ne
         stage = DEEPSTALL_STAGE_WAIT_FOR_BREAKOUT;
         //compute optimal path for landing
         build_approach_path();
+        loiter_sum_cd = 0; // reset the loiter counter
         // no break
         }
     case DEEPSTALL_STAGE_WAIT_FOR_BREAKOUT:
         if (!verify_breakout(current_loc, arc_entry, height)) {
+            int32_t target_bearing = landing.nav_controller->target_bearing_cd();
+            int32_t delta = wrap_180_cd(target_bearing - last_target_bearing);
+            if (delta > 0) { // only accumulate turns in the correct direction
```

Comment:
You know, I'd like to hope that it would be, since thats the same thing used 15 lines up, but I've never tried a CCW deepstall, and I suspect your right that it goes wrong... Separate issue though since you won't even get to this line...